shell/platform/windows/cursor_handler.cc

```diff
@@ -11,9 +11,22 @@
 static constexpr char kChannelName[] = "flutter/mousecursor";
 
 static constexpr char kActivateSystemCursorMethod[] = "activateSystemCursor";
-
 static constexpr char kKindKey[] = "kind";
 
+// [kSetCustomCursorMethod] allows developers to set a custom cursor with
+// rawRGBA buffer.
+static constexpr char kSetCustomCursorMethod[] = "setCustomCursor";
+// std::vector<uint8_t> value for custom cursor rawRGBA buffer.
+static constexpr char kCustomCursorBufferKey[] = "buffer";
+// double value for the hotx value of custom cursor.
+static constexpr char kCustomCursorHotxKey[] = "hotx";
```

Comment:
```suggestion
static constexpr char kCustomCursorHotXKey[] = "hotX";
```
Use camel case for key names.